pingouin/nonparametric.py

```diff
@@ -222,16 +225,20 @@ def mwu(x, y, alternative='two-sided', **kwargs):
         Association and the American Statistical Association, 25(2),
         101–132. https://doi.org/10.2307/1165329
 
+    .. [5] Campbell, M. J. & Gardner, M. J. (1988). Calculating confidence 
+           intervals for some non-parametric analyses.
+           British Medical Journal Volume 226, 1988.
+
     Examples
     --------
     >>> import numpy as np
     >>> import pingouin as pg
     >>> np.random.seed(123)
     >>> x = np.random.uniform(low=0, high=1, size=20)
     >>> y = np.random.uniform(low=0.2, high=1.2, size=20)
-    >>> pg.mwu(x, y, alternative='two-sided')
-         U-val alternative    p-val    RBC    CLES
-    MWU   97.0   two-sided  0.00556  0.515  0.2425
+    >>> pg.mwu(x, y, alternative='two-sided',confidence=0.95)
+         U-val alternative    p-val    RBC    CLES     CI95%
+    MWU   97.0   two-sided  0.00556  0.515  0.2425     [-0.39290395101879694, -0.09400270319896187]
```

Comment:
Is this the actual output that you get? The CI should normally be rounded to two decimals by the _postprocess_dataframe function

Reply:
That's the actual output I get. I was wondering about that, too. But then again, the t-test also gives me full floats (at least when confidence!=0.95), so I thought that was intentional.
I can of course round it in MWU or do you want to adress that elsewhere?